holoviews/plotting/bokeh/raster.py

```diff
@@ -92,9 +103,10 @@ def on_change(attr, old, new):
             if np.isinf(new).all():
                 return
             data_sel = self._hover_data.sel(**dict(zip(self._hover_data.coords, new)), method="nearest").to_dict()
-            # TODO: When ValueOf support formatter remove the rounding
-            # https://github.com/bokeh/bokeh/issues/14123
-            data_coords = {dim: round(data_sel['coords'][dim]['data'], 3) for dim in coords}
+            if BOKEH_GE_3_7_0:
+                data_coords = {dim: data_sel['coords'][dim]['data'] for dim in coords}
+            else:
+                data_coords = {dim: round(data_sel['coords'][dim]['data'], 3) for dim in coords}
```

Comment:
It would be OK with me to only have the `BOKEH_GE_3_7_0` code.

Reply:
What's the behavior for older bokeh versions?

Reply:
This is only for the x,y-coords, so it would show `1.999999999999` instead of `2.0`. 

Though, I'm unsure how it would work if x,y-coords are datetime.

Reply:
It errored out when we had datetime as x,y coords. I have fixed this in https://github.com/holoviz/holoviews/pull/6531/commits/3ae5e869f34dc026dead22e661f463bb13dfd447 and also removed the round here.